extensions/Worker.Extensions.Http.AspNetCore/src/HostBuilderExtensions.cs

```diff
@@ -0,0 +1,38 @@
+﻿// Copyright (c) .NET Foundation. All rights reserved.
+// Licensed under the MIT License. See License.txt in the project root for license information.
+
+using System;
+using System.Collections.Generic;
+using System.Linq;
+using System.Text;
+using System.Threading.Tasks;
+using Microsoft.AspNetCore.Hosting;
+using Microsoft.Azure.Functions.Worker.Core.Pipeline;
+using Microsoft.Extensions.Hosting;
+
+namespace Microsoft.Azure.Functions.Worker.Extensions.Http.AspNet
+{
+    public static class HostBuilderExtensions
+    {
+        public static IHostBuilder ConfigureAspNetIntegration(this IHostBuilder builder)
+        {
+            var port = Environment.GetEnvironmentVariable("Azure_Functions_HttpProxyingPort");
```

Comment:
I was realizing that we kind of already have a precedent here of using `FUNCTIONS_` as the prefix for all of our special env vars -- suggest changing this (and making it all-caps).

Reply:
Yes, we should. I'm leaving some of this logic intact for now (restructuring things only) and will follow up with additional changes in different areas.